src/nmo.js

```diff
@@ -49,6 +49,8 @@ nmo.load = function load (opts) {
       .load(opts)
       .then((config) => {
         nmo.config = config;
+        nmo.config.set('name', 'nmo');
+        nmo.config.set('help-directory', `${__dirname}/..`);
```

Comment:
not sure if we need to migrate old other config. the `ini` module should make migration a 4-5 liner